Classes/Services/ElasticSearch/ElasticSearch.php

```diff
@@ -0,0 +1,311 @@
+<?php
+namespace EWW\Dpf\Services\ElasticSearch;
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+use Elasticsearch\ClientBuilder;
+use EWW\Dpf\Domain\Workflow\DocumentWorkflow;
+use EWW\Dpf\Helper\ElasticsearchMapper;
+use TYPO3\CMS\Extbase\Object\ObjectManager;
+use EWW\Dpf\Configuration\ClientConfigurationManager;
+use EWW\Dpf\Domain\Model\Document;
+use EWW\Dpf\Helper\Mods;
+
+class ElasticSearch
+{
+    /**
+     * frontendUserHelper
+     *
+     * @var \EWW\Dpf\Helper\FrontendUserHelper
+     * @inject
+     */
+    protected $frontendUserHelper = null;
+
+
+    protected $client;
+
+    protected $server = 'host.docker.internal'; //127.0.0.1';
```

Comment:
[code] Do we still have a plugin configuration section for the Elasticsearch connection parameters?

Reply:
Yes, it still has to be configured within the client dataset in the list module.